core/Dockerfile

```diff
@@ -19,6 +19,14 @@ RUN /home/pi/tools/install-static-binaries.sh
 # BlueOS-docker base image
 FROM bluerobotics/blueos-base:v0.1.0
 
+# Define arguments for target platform
+ARG TARGETARCH
```

Comment:
Can we add a comment about how this arguments are generated by --platform ?

```
# These arguments are defined automatically by buildx when using `--platform`
```

Reply:
done